guides/common/modules/ref_glossary-of-terms-used-in-project.adoc

```diff
@@ -299,27 +457,55 @@ Role:: Specifies a collection of permissions that are applied to a set of resour
 Roles can be assigned to users and user groups.
 {Project} provides a number of predefined roles.
 
+ifndef::satellite[]
+[[Salt]]
+Salt::
+Salt is a configuration management tool used to maintain hosts in certain defined states, for example have packages installed or services running.
+It is designed to be idempotent.
+For more information about using Salt to configure hosts, see {ManagingConfigurationsSaltDocURL}[{ManagingConfigurationsSaltDocTitle}].
+endif::[]
+
 [[SCAP_content]]
-SCAP content:: A file containing the configuration and security baseline against which hosts are checked.
-Used in compliance policies.
+SCAP content::
+SCAP stands for Security Content Automation Protocol and refers to `.xml` files containing the configuration and security baseline against which hosts are checked.
+{Project} uses SCAP content in compliance policies.
 
 ifndef::orcharhino,satellite[]
-[[SmartProxy]]
-{SmartProxy}:: A {SmartProxy} component that can integrate with external services, such as *DNS* or *DHCP*.
-{SmartProxies} are useful for {Project} deployment across various geographical locations.
+[[Smart-Proxy]]
+include::snip_glossery-term-smart-proxy.adoc[]
 endif::[]
 
 [[Subnet_image]]
 Subnet image:: A type of generic image for PXE-less provisioning that communicates through {SmartProxyServer}.
 
 [[Subscription]]
+// TODO: Rename to "Red Hat subscription"?
 Subscription:: An entitlement for receiving content and service from Red{nbsp}Hat.
 
+[[Subscription-Manager]]
+Subscription Manager::
+`subscription-manager` is a client application to register hosts to {Project}.
+`subscription-manager` uses xref:Activation_key[activation keys] to consume content on hosts.
```

Comment:
subscription-manager can also set system purpose attributes and content overrides from its CLI; not sure if that's worth mentioning here.

Reply:
In the spirit of the glossary, a short explanation what subscription-manager means is enough. Everything else should be covered in concepts and procedures. ACK @asteflova ?